pkg/daemon/ceph/osd/agent.go

```diff
@@ -142,26 +145,32 @@ func (a *OsdAgent) removeDirs(context *clusterd.Context, removedDirs map[string]
 	return nil
 }
 
-func (a *OsdAgent) configureDevices(context *clusterd.Context, devices *DeviceOsdMapping) ([]oposd.OSDInfo, error) {
-	var osds []oposd.OSDInfo
-	if devices == nil || len(devices.Entries) == 0 {
+func (a *OsdAgent) configureAllDevices(context *clusterd.Context, devices *DeviceOsdMapping) ([]oposd.OSDInfo, error) {
+
+	// prepare the OSDs with ceph-volume
+	osds, configured, err := a.configureDevices(context, devices)
+	if err != nil {
+		return nil, fmt.Errorf("failed to configure devices with ceph-volume. %+v", err)
+	}
+
+	if devices == nil || len(devices.Entries) == 0 || configured {
 		return osds, nil
 	}
 
+	// get info about legacy OSDs prepared without ceph-volume
```

Comment:
Is this a "placeholder" or a comment hinting at what the next n lines are doing?